### DISCUSSION TITLE
KAFKA UI hung on kafka-connect tab

### DISCUSSION THREAD

**Discussion by sai9920** [provectus/kafka-ui#4443] [Q&A]
We have added multiple kafka-connect endpoints, but few times it loads for longer and timeout, restart of the pod would fix the issue. But this is happening very frequent. Using 83b5a60 v0.7.2 version
https://github.com/provectus/kafka-ui/releases

**Comment by germanosin** [comment]
Me and @Haarolean  no longer support this repo, don't you mind to post this to https://github.com/kafbat/kafka-ui
